app/src/main/java/com/example/stromtracker/database/AppDatabase.kt

```diff
@@ -41,21 +44,39 @@ abstract class AppDatabase : RoomDatabase() {
             ).addCallback(object : Callback() {
                 override fun onCreate(db: SupportSQLiteDatabase) {
                     super.onCreate(db)
-                    Thread(Runnable { prepopulateDb(context, getInstance(context)) }).start()
+                    Thread(Runnable { prepopulateDb(context, getInstance(context)) }
+                    ).start()
+
                 }
             })
                 .build()
 
         }
-         fun destroyInstance() {
+
+        fun destroyInstance() {
             INSTANCE = null
         }
 
         fun prepopulateDb(context: Context, db: AppDatabase) {
-            db.haushaltDao().insertHaushalt(Haushalt("name", 0.0, 1, 0.0, null, false), Haushalt("test2", 0.0, 1, 0.0, null, false))
+            db.haushaltDao().insertHaushalt(
+                Haushalt("Haushalt1", 0.0, 1, 0.0, null, false)
+            )
+            db.raumDao().insertRaum(Raum("Sonstiges", 1))
```

Comment:
Wenn, dann sollten wir auch noch die anderen zwei Standardräume erstellen

Reply:
Es gibt zwei Standardräume?

Reply:
Hab ich noch in Räume. Muss aber noch gemacht fertig werden